File: nodejs/test/utils.test.js
const { cleanText, slugify } = require('../src/utils');

test('cleanText removes punctuation', () => {
  expect(cleanText('Hello, World!')).toBe('hello world');
});

test('slugify replaces spaces', () => {
  expect(slugify('Hello World', 'English')).toBe('hello-world');
});

File: go/pkg/utils_test.go
package pkg

import "testing"

func TestCleanTextRemovesPunctuation(t *testing.T) {
    got := CleanText("Hello, World!")
    want := "hello world"
    if got != want {
        t.Errorf("got %q, want %q", got, want)
    }
}

func TestSlugifySpaces(t *testing.T) {
    got := Slugify("Hello World", "English")
    want := "hello-world"
    if got != want {
        t.Errorf("got %q, want %q", got, want)
    }
}